### PR TITLE
Update pulumiup 2024 page

### PR DESCRIPTION
**Pull request by sean1588**
issue: https://github.com/pulumi/marketing/issues/1108

### CODE REVIEW

File: content/pulumi-up/_index.md [hunk 1]
```diff
@@ -189,16 +255,31 @@ speakers:
     title: Developer at Avanade
```
Comment (SaraDPH) [inline content/pulumi-up/_index.md:255]:
```suggestion
    title: Software Engineer
```

**Comment by SaraDPH** [comment]
Wow, it looks really nice, @sean1588 !
Can you move the workshops up, before "Featured Partners"?

**Review by SaraDPH — APPROVED** [review]

**Comment by SaraDPH** [comment]
@sean1588 the add to schedule links actually changed depending on the panel. But if all the buttons need to have the same link, then send them to https://conference.pulumi.com/schedule/?utm_source=PulumiUp&utm_medium=web&utm_campaign=FY2025Q1_Event_PulumiUP

**Comment by sean1588** [comment]
> @sean1588 the add to schedule links actually changed depending on the panel. But if all the buttons need to have the same link, then send them to https://conference.pulumi.com/schedule/?utm_source=PulumiUp&utm_medium=web&utm_campaign=FY2025Q1_Event_PulumiUP

ahh thanks for catching that, didn't notice the links were different. I got them updated!

**Comment by SaraDPH** [comment]
Looks great!

**Review by shughes26 — APPROVED** [review]

**Review by cnunciato — APPROVED** [review]

**Pull request merged**